data/changelog.txt

```diff
@@ -380,6 +387,19 @@ Fix Brackenwight special attack message
 [Magiclysm] Last Mage on Earth start
 Tamable Wildlife: add Cougars to list of tameables
 [Magiclysm] Add spellcasting proficiencies.
+[MoM] Channeling powers costs focus
+Edit spell UI word choice when using psionics
+Fixes spellcasting proficiencies adding when they should be subtracting
+[MoM] High levels of weariness turn off your powers
+More specific messages for being unable to use supernatural powers
+[Xedra Evolved] New Arvore spell--Verdant Imprisonment
+Tune channeling spells with scale modifier.
+[MoM] Edit portal storm conditions to support moddability, use that for Telepathic Shield in MoM
+allow incandescent lance to repair metal items
+XE: Trait to avoid learning dream magick
+Too much pain shuts down your powers
+Tune evocation spells with scale modifier.
+[MoM] Add luminance to Eaters
```

Comment:
```suggestion
Edit spell UI word choice when using psionics
More specific messages for being unable to use supernatural powers
XE: Trait to avoid learning dream magick
Too much pain shuts down your powers
```